pkg/sync/v5/translate_test.go

```diff
@@ -0,0 +1,77 @@
+package sync
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestTranslatableData(t *testing.T) {
+	var errors []string
+
+	err := filepath.Walk("data", func(path string, info os.FileInfo, errIn error) error {
+		if errIn != nil {
+			return errIn
+		}
+
+		if info.IsDir() || !strings.HasSuffix(info.Name(), ".yaml") {
```

Comment:
prob should remove the .yaml suffix check and not allow any other files in there